ItemRack/ItemRackQueue.lua

```diff
@@ -85,7 +85,7 @@ end
 
```

Comment:
Please add a verb to the commit message and start with a capital letter.

Reply:
Should be sorted